Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM ubuntu:18.04
-RUN apt update && \
-    apt -y --no-install-recommends install \
-        python2.7-minimal python-pip python-setuptools python-natsort python-repoze.lru \
-        git language-pack-de rsync && \
-    pip install oerplib
-RUN git clone https://github.com/fau-fablab/oerp-pricelist /oerp-pricelist
-WORKDIR /oerp-pricelist
-RUN pip install --upgrade -r requirements.txt
-COPY config.ini config.ini
+# See https://hub.docker.com/r/frolvlad/alpine-python2/dockerfile
+FROM alpine:3.14
```

Comment:
Our production infrastructure only uses Ubuntu due to the longer LTS support and because we already have the tooling to check for package updates and automatically rebuild the containers. So the standard Dockerfile should remain Ubuntu. Feel free to add an alternative `Dockerfile.alpine` as a separate file.

Reply:
I'm not sure that I follow. Does it matter what is going on inside the container? Does our infrastructure run some sort of introspection, examining in-container data?
I was simply going for Alpine because it builds faster and is much lighter on resources when running than a full blown Ubuntu.
I don't care that much either way, though.

Reply:
To put it simply: We use Ubuntu for everything else, so for the sake of maintainability it makes no sense to have a special exception just for one container.

Yes, we execute in-container commands to check for package updates, and we use a custom Ubuntu base image for everything. So we currently can not use anything else.

Reply:
I see! I assumed we were treating docker images as a black box. Doing introspection changes things of course!

Reply:
@mgmax How do I access our custom ubuntu image? Are we running a registry server? What's its name?

Reply:
No, there is no registry; it is directly built on the server. https://github.com/fau-fablab/brain-docker-config/ , see `basislinux*`.